index.html

```diff
@@ -316,19 +316,19 @@ <h2 class="inter">
 			<hr>
 			<ul class="list-icons dark animate">
 				<li>
-					<a href="#"><img src="Works-Images/Github dark.png"></a>
+					<a href="#"><img src="Works-Images/Github dark.png" alt="Github Logo"></a>
 				</li>
 				<li>
-					<a href="#"><img src="Works-Images/Linkedin dark.png"></a>
+					<a href="#"><img src="Works-Images/Linkedin dark.png" alt="Linkedin Logo"></a>
 				</li>
 				<li>
-					<a href="#"><img src="Works-Images/Peace dark.png"></a>
+					<a href="#"><img src="Works-Images/Peace dark.png" alt="Angellist Logo"></a>
```

Comment:
- [ ] Please remove the text "logo", even don't use the words like `button`, `icon`, `image of` and `logo` in your **alternative-text** because by default when  a person who use screen-readers, screen reader automatically provide the context. Kindly use "Angelist Profile" because it will assist the person to know whether he/she is interested to look into your `profile` or not.